cpu/cpu_linux.go

```diff
@@ -141,9 +141,13 @@ func InfoWithContext(ctx context.Context) ([]InfoStat, error) {
 			c.CPU = int32(t)
 		case "vendorId", "vendor_id":
 			c.VendorID = value
+		case "CPU implementer":
+			if value == "0x41" {
```

Comment:
Could you add more implementers according to [ARM document](https://developer.arm.com/documentation/ihi0014/q/Programmers--Model/Detailed-register-descriptions/ID-Register--ETMIDR--ETMv2-0-and-later) and [linux code](https://github.com/torvalds/linux/blob/master/arch/arm64/include/asm/cputype.h#L54)?
```
// from https://github.com/torvalds/linux/blob/master/arch/arm64/include/asm/cputype.h#L54
switch value {
  case "0x41":
     c.VendorID = "ARM"
  case "0x50":
     c.VendorID = "APM"
.....
(snip)
  case "0x61":
     `c.VndorID` = "APPLE"  
}
```